blocks/footer/footer.css

```diff
@@ -102,15 +105,18 @@ footer ul li {
 }
 
 .ft-social .ft-social-list:nth-child(2)>ul {
-  display: inline-flex;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px ;
 }
 
 .ft-social .ft-social-list:nth-child(2)>ul>li:not(:first-child) {
-  padding: 0 0 0 28px;
+  /* padding: 0 0 0 28px; */
 }
 
 .ft-social .ft-social-list:last-child {
   margin: auto 0;
+  margin-left: auto;
```

Comment:
This will move the third element tot he right when it wraps. Can you do `margin-right: auto` to the second child instead?